lib/builder-webpack4/src/preview/base-webpack.config.ts

```diff
@@ -107,15 +110,19 @@ export async function createDefaultWebpackConfig(
           test: /\.(svg|ico|jpg|jpeg|png|apng|gif|eot|otf|webp|ttf|woff|woff2|cur|ani|pdf)(\?.*)?$/,
           loader: require.resolve('file-loader'),
           options: {
-            name: 'static/media/[name].[hash:8].[ext]',
+            name: isProd
+              ? 'static/media/[name].[contenthash:8].[ext]'
+              : 'static/media/[path][name].[ext]',
```

Comment:
we need a path here for handling assets with the same name in dev mode.

So previously `some/file.svg` and `another/file.svg` became `static/media/file.svg` and it led to conflict

After this change we will have 2 assets:
`some/file.svg` → `static/media/some/file.svg`
`another/file.svg` → `static/media/another/file.svg`